devicetypes/smartthings/zll-switch.src/zll-switch.groovy

```diff
@@ -0,0 +1,124 @@
+/**
+ *  Copyright 2016 SmartThings
```

Comment:
Update copyright date

Reply:
should be 2017 based on the date it was created